From 34694c7d927e1ac9ff92db95136ebc0a7859e5d5 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Fri, 26 May 2023 14:05:40 +0200
Subject: [PATCH] fstab-generator: use correct swap name var

Follow-up to 9445623363.

(cherry picked from commit 3aed25932891d447b5a7ceec6ab61813c06fb0ec)

Related: #2190226
---
 src/fstab-generator/fstab-generator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fstab-generator/fstab-generator.c b/src/fstab-generator/fstab-generator.c
index 910e29d26b..b92fb40f9b 100644
--- a/src/fstab-generator/fstab-generator.c
+++ b/src/fstab-generator/fstab-generator.c
@@ -811,7 +811,7 @@ static int parse_fstab_one(
 
         is_swap = streq_ptr(fstype, "swap");
         if (is_swap && use_swap_enabled && !arg_swap_enabled) {
-                log_info("Swap unit generation disabled on kernel command line, ignoring swap entry for %s.", what);
+                log_info("Swap unit generation disabled on kernel command line, ignoring swap entry for %s.", what_original);
                 return 0;
         }
 
